CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 ### Added
 
+- [#3967](https://github.com/firecracker-microvm/firecracker/pull/3967/):
```

Comment:
commit message: please specify what part of documentation is being updated. With `git log --oneline` it's hard to tell.

Reply:
no, just mention vcpu configuration somehow

Reply:
Added `(cpu templates)` to the commit header.